### ISSUE TITLE
Signal proposals: Make the signal text more obvious somehow

### ISSUE THREAD

**Issue by mochet** [Joystream/pioneer#4732]
# Problem
I often engage with people in chat who ask about the progress of the project and send them things like the council term plan signal proposals (example: https://pioneerapp.xyz/#/proposals/preview/727) however one problem I have is that it doesn't feel obvious enough on how to access the `"Signal Text"` and therefore I am not sure how useful it even is to send this to people (keep in mind this is sending a `Signal proposal` to a complete outsider to our project--these users may not speak English as a first language and likely have little understand/care of the relevance of `signal`, `rationale` + `description`... they are being sent to this page to access the actual signal text as a priority).

![image](https://github.com/Joystream/pioneer/assets/50830161/aa25fd2d-589d-4e3d-9c9d-a14b14701f61)
These new users encounter this huge amount of information and I do not think it is obvious enough on how to see the actual Signal Text--which is literally the _most important_ point of them accessing this page.

# Solutions
I can think of a few but am open to ideas here:
* Show a preview text/truncated section (similar to how new forum threads have a preview at the top of the forum page)
* Add a way to direct link to the `signal text` itself
* Have the signal text show truncated (with an option to show full text) inline on the page rather than in a sidebar.

**Comment by kdembler** [comment]
Agreed with this. Visually "Rationale" stands out more and is given more prominence when it should be the other way around. It would be good to get some designer involved

**Comment by traumschule** [comment]
- [ ] #4719 Option to link to #signal 
- [ ] Make Signal expandable like Rationale
- [ ] automatically expand it for #signal
